code/modules/telesci/gps.dm

```diff
@@ -34,6 +34,8 @@ var/list/GPS_list = list()
 /obj/item/device/gps/AltClick(mob/user)
 	if(!in_range(src, user))
 		return
+	if(!user.canUseTopic(user))

```

Comment:
I think they're actually incorrect, and in all cases it should pass through the item it's attempting to use.

Nothing actually uses it now but stuff may want to use it later.